.goreleaser.yaml

```diff
@@ -0,0 +1,36 @@
+project_name: grid-intensity
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
+      - darwin
+archives:
+  - replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      386: i386
+      amd64: x86_64
+brews:
+  - tap:
+      owner: thegreenwebfoundation
+      name: homebrew-grid-intensity
+      token: {{ .Env.HOMEBREW_TAP_GITHUB_TOKEN }}
```

Comment:
See https://github.com/goreleaser/goreleaser/issues/2026 and https://github.com/goreleaser/goreleaser/issues/2027

The default setup is to use a PAT but with repo scope it can push to all repos the user has access to.

The alternative is to create the token for a GitHub App which can be scoped to just the homebrew-grid-intensity repo.

But I don't seem to have permission to create GitHub Apps. So @mrchrisadams I think I'm going to need some help here.

Reply:
cripes - how do we make github apps again?

oh yeah, the docs are here.
https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app

I'll have a look through to see how we can do this

Reply:
ok, I've figured out how to create a github app. I've created the _Grid Intensity App O Tron_ and invited you to it. you should be able to manage it, and make any changes.

If you need to chat about anything there that shouldn't be public, please DM me in Zulip 👍